.circleci/config.yml

```diff
@@ -41,7 +45,24 @@ jobs:
           command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > .npmrc
       - run:
           name: Publish package
-          command: npm publish --access public
+          command: |
+            PACKAGE_VERSION=$(cat ./package.json \
```

Comment:
nit: would this works? `NPM_PACKAGE_VERSION=$(npm pkg get version)`

Reply:
Would you look at that, it totally does! If you don't mind, I'll leave as is. The only difference is that it's in `"quotes"` which would probably still work, but I already tested this version on CI.

![image](https://user-images.githubusercontent.com/3044528/236474442-ff2b2e62-5a2a-446e-9729-ff9d566c3636.png)